content/terraform/v1.14.x (alpha)/docs/cli/commands/query.mdx

```diff
@@ -0,0 +1,86 @@
+---
+page_title: locals block reference for `.tfquery.hcl` files
+description: |-
+ Add the `locals` block to your `.tfquery.hcl` files define values to reuse within the local Terraform module. 
+---
+
+# `terraform query` command
+
+The `terraform query` command queries existing infrastructure for unmanaged resources according to the `tfquery.hcl` file so that you can import them into your Terraform workspace.
```

Comment:
```suggestion
The `terraform query` command queries existing infrastructure for resources according to the `tfquery.hcl` file so that you can import them into your Terraform workspace.
```